### DISCUSSION TITLE
Combine put and patch routes to the same handler

### DISCUSSION THREAD

**Discussion by 3olkin** [rwf2/Rocket#1806] [Questions] [answered]
Couldn't find similar issue or discussion.
First of all, I'm new to rocket and need some help. Is there a way to combine Put and Patch routes to the same handler? 

Personally, I found  this workaround, but it doesn't look good, so I believe that better solution exists. 
![image](https://user-images.githubusercontent.com/58516282/128053670-7de60f73-f07a-4889-b27f-b39ae61bf3cd.png)

**Comment by jebrosen** [comment, marked as answer]
This is actually one of Rocket's oldest feature ideas! (#2)

But this also looks incorrect in this particular case. PUT and PATCH have different semantics, are not intended for the same use cases, and may be treated differently by browsers and caches. Implementing both to do the same thing seems unnecessary, and could have negative side effects.

**Reply by 3olkin** [reply]
thx for your answer
